src/Symfony/Component/Translation/Dumper/XliffFileDumper.php

```diff
@@ -36,6 +48,10 @@ public function dump(MessageCatalogue $messages, $options = array())
             $this->defaultLocale = \Locale::getDefault();
         }
 
+        if (array_key_exists('tool_info', $options)) {
+            $this->toolInfo = array_merge($this->toolInfo, $options['tool_info']);
```

Comment:
btw, this is precisely the reason why we introduced the new ``formatCatalogue`` method to be able to access options while dumping (see the YamlFileDumper)